pattern-library/js/select-replace.js

```diff
@@ -12,18 +12,16 @@ define([
      * Choosing an option in the replacement menu also updates the
      * default select menu thus maintaining accessibility.
      */
-
-
     var CustomSelectReplacement = {
 
         vars: {
-            replaced:       $('.replace-select'),
-            replacedClass:  'is-replaced is-transparent',
-            customClass:    'wrapper-custom-select',
-            wrapperClass:   'wrapper-replace-select',
-            valueClass:     'replace-value',
-            iconClass:      'icon-caret-down',
-            hoverClass:     'is-hover'
+            replaced: $('.replace-select'),
```

Comment:
FYI. I remove the $ in tabs.js, so that all is consistent. Sorry @AlasdairSwan , if I have to pick, I'd rather edit fewer files, since it is only tabs.js has this.